project.clj

```diff
@@ -65,15 +65,14 @@
                  ;; bridge to allow some spring/activemq stuff to log over slf4j
                  [org.slf4j/jcl-over-slf4j "1.7.10"]
                  ;; WebAPI support libraries.
-                 [net.cgrand/moustache "1.1.0" :exclusions [ring/ring-core org.clojure/clojure]]
-                 [compojure "1.3.3"]
+                 [compojure "1.4.0"]
```

Comment:
Not yet. I forgot that we still use some stuff from compojure. pdb-routing.clj is a good example. I don't see anything we're using compojure for that wouldn't be pretty easy to move over to comidi. Once we've done that, we'll have no code that references compojure and I'll feel comfortable removing it.